ops/log.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, model_backend, level=logging.DEBUG):
         self.model_backend = model_backend
 
     def emit(self, record):
+        """Send the specified logging record to the Juju backend."""
```

Comment:
Here I'd point at `logging.Handler` also.

Reply:
Sorry, I merged this without checking these comments, but will add these improvements to next branch.

What do you mean about pointing it to logging.Handler?

Reply:
Well, this `emit()` method is not called by us anywhere, and while we do implement and call a _different_ `emit()` method for our own events it is not related in any way to this one. This one only works because this class implements `logging.Handler`, and it's the logging infra that calls emit itself.
So somehow pointing that out is probably a good idea.

Reply:
Ah, got it, thanks! Added to the next branch.